RELEASE_NOTES

```diff
@@ -6,9 +6,6 @@
 
  * Build Issues
    - CH4 will not build with Solaris compilers.
```

Comment:
The master currently does build with Solaris : https://jenkins-pmrs.cels.anl.gov/job/mpich-master-ch4-ofi/

It only build with external libfabric and external ucx though.

Reply:
OK, great. I'll not that and the limitations.